shared/cfg/guest-hw.cfg

```diff
@@ -352,6 +357,10 @@ variants image_backend:
         nfs_mount_src = localhost:/mnt/nfs_export
         setup_local_nfs = yes
         export_options = rw
+    - nvme_direct:
```

Comment:
do we need a 'image_raw_device = yes' here ?

Reply:
Thanks, have updated.